packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js

```diff
@@ -40,16 +42,28 @@ const ContentItem = ({ copy, heading, image }) => (
         data-autoid={`${stablePrefix}--content-item__copy`}
         className={`${prefix}--content-item__copy`}
         dangerouslySetInnerHTML={{
-          __html: markdownToHtml(copy),
+          __html: markdownToHtml(copy, { bold: false }),
```

Comment:
I realize this is not apart of this issue, but I just want to point out #1256 is still outstanding for this sub-pattern so we keep it fresh in the forefront.

Reply:
@kennylam already completed this.